server/app.ts

```diff
@@ -36,11 +37,12 @@ export default function createApp(services: Services): express.Application {
   app.use(setUpStaticResources())
   nunjucksSetup(app, services.applicationInfo)
   app.use(setUpAuthentication())
-  app.use(authorisationMiddleware())
+  app.use(authorisationMiddleware(['ADJUSTMENTS_MAINTAINER']))
   app.use(setUpCsrf())
   app.use(setUpCurrentUser(services))
   app.get('*', getFrontendComponents(services))
   app.use('/:nomsId', populateCurrentPrisoner(services.prisonerSearchService))
+  app.use('*', supportUserReadonlyMiddleware())
```

Comment:
Out of curiosity does 
```suggestion
  app.post('*', supportUserReadonlyMiddleware())
```
work then you can remove the method check in the middleware?

Reply:
I'll have a test.